### PR TITLE
Internal improvement: Clean up some scenario tests

### PR DESCRIPTION
**Pull request by eggplantzzz**
This PR primarily removes the `Reporter` class test helper. It seems to mostly be unused and so it would simplify setups to have it gone from the codebase. Other than that it consolidates some test setups and a few other miscellaneous things.

### CODE REVIEW

File: packages/truffle/test/scenarios/commands/console.js [hunk 1]
```diff
@@ -7,20 +7,17 @@ const sandbox = require("../sandbox");
 const tmp = require("tmp");
 
 describe("truffle console", () => {
-  let config, project;
+  let config;
   const logger = new MemoryLogger();
+  const project = path.join(__dirname, "../../sources/console");
```
Comment (sukanyaparashar) [inline packages/truffle/test/scenarios/commands/console.js:12]: Yeah! That should have been a `const`.

File: packages/truffle/test/scenarios/commands/console.js [hunk 2]
```diff
@@ -7,20 +7,17 @@ const sandbox = require("../sandbox");
...
 
-  before(async () => await Server.start());
-  after(async () => await Server.stop());
-
-  project = path.join(__dirname, "../../sources/console");
-
-  before(async function () {
-    this.timeout(10000);
+  before(async () => {
```
Comment (sukanyaparashar) [inline packages/truffle/test/scenarios/commands/console.js:14]: Makes sense to use `before` only once here.

**Review by sukanyaparashar — APPROVED** [review]
This looks good to me !

**Review by cliffoo — APPROVED** [review]
Nice one @eggplantzzz !

**Pull request merged**